plugins/MauticCrmBundle/Entity/PipedriveDeletion.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace  MauticPlugin\MauticCrmBundle\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
+
+class PipedriveDeletion
+{
+    /**
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $objectType;
+
+    /**
+     * @var int
+     */
+    private $integrationEntityId;
+
+    /**
+     * @var int
+     */
+    private $deletedDate;
+
+    public static function loadMetadata(ORM\ClassMetadata $metadata)
+    {
+        $builder = new ClassMetadataBuilder($metadata);
+        $builder->setTable('plugin_crm_pipedrive_deletions');
+
+        $builder->addId();
```

Comment:
Are you sure the INT is good enough? Consider using BIGINT. I don't know how many records to expects in this table.

We also find pretty good to set properties required for the entity in the constructor. Is there such property that saving this without it doesn't make sense? As none of the columns is nullable I guess it's all of them?